app/views/admin/users/index/_tabs.html.erb

```diff
@@ -7,10 +7,10 @@
       <%= link_to "Invited Members", admin_invitations_path, class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/invitations'}", "aria-current": ("page" if params[:controller] == "admin/invitations") %>
     </li>
     <li>
-      <%= link_to admin_users_gdpr_delete_requests_path, class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/users/gdpr_delete_requests'}", "aria-current": ("page" if params[:controller] == "admin/users/gdpr_delete_requests") do %>
+      <%= link_to admin_gdpr_delete_requests_path, class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/users/gdpr_delete_requests'}", "aria-current": ("page" if params[:controller] == "admin/users/gdpr_delete_requests") do %>
```

Comment:
```suggestion
      <%= link_to admin_gdpr_delete_requests_path, class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/gdpr_delete_requests'}", "aria-current": ("page" if params[:controller] == "admin/gdpr_delete_requests") do %>
```
This tweak should restore the current tab highlighting

Reply:
Done ✅